### PR TITLE
Feat/related articles using Slice and add Lead and Two

### PR DESCRIPTION
**Pull request by colincclark**
- adds Lead and Two template to related articles
- integrates new Slice component with Related Articles
- refactors Related Articles for consistency with other packages
- fixes layout issues, namely, image/summary show/hide logic and responsive headline in Related Articles and lead and two layout widths in Slice
- improves Related Articles test names
- removes the old version of Slice

NB: this is iterative. We are not using Card in Related Articles on this PR

### LEAD_AND_TWO (1 article)
lead_1

### LEAD_AND_TWO (2 articles)
lead_2

### LEAD_AND_TWO (3 articles)
lead_3

### LEAD_AND_TWO (1 article)
screen shot 2018-03-07 at 12 42 43

### LEAD_AND_TWO (2 articles)
screen shot 2018-03-07 at 12 42 55

### LEAD_AND_TWO (3 articles)
screen shot 2018-03-07 at 12 43 05

### LEAD_AND_TWO (1 article)
screen shot 2018-03-07 at 12 29 50

### LEAD_AND_TWO (2 articles)
screen shot 2018-03-07 at 12 32 47

### LEAD_AND_TWO (3 articles)
screen shot 2018-03-07 at 12 33 11

### CODE REVIEW

File: packages/article/article.stories.js [hunk 1]
```diff
@@ -220,33 +223,54 @@ storiesOf("Pages/Article", module)
...
     />
   ))
-  .add("Three related articles", () => (
+  .add("Default template with three related articles", () => (
     <RelatedArticles
-      {...createRelatedArticlesProps(threeRelatedArticlesFixture.data)}
+      {...createRelatedArticlesProps(defaultThreeRelatedArticlesFixture.data)}
+    />
+  ))
+  .add("Lead and two template with one related article", () => (
```
Comment (aronshamash) [inline packages/article/article.stories.js]: Should this be the actual template name ?

File: packages/slice/templates/lead/responsive.web.js [hunk 2]
```diff
@@ -98,14 +97,18 @@ export const getLeadContainer = ({ hasSupports }) => {
       padding-right: 0;
     `;
       const withSupportsStyle = `
-      flex-grow: 3;
+      flex-basis: 0 !important;
+      flex-grow: ${supportCount === 1 ? "3" : "1.75"};
```
Comment (aronshamash) [inline packages/slice/templates/lead/responsive.web.js]: 1.75 should go in a const

Comment (colincclark) [reply packages/slice/templates/lead/responsive.web.js]: it only applies here...the other one, on line 110 is not the same value

File: packages/article/article.stories.js (hunk 1 above)
Comment (colincclark) [reply packages/article/article.stories.js]: Not sure what you mean? Lead and two is the template name. Do you mean the story should be something like "LEAD_AND_TWO with only lead"?

File: packages/slice/templates/lead/responsive.web.js (hunk 2 above)
Comment (colincclark) [reply packages/slice/templates/lead/responsive.web.js]: resolved

File: packages/article/article.stories.js (hunk 1 above)
Comment (colincclark) [reply packages/article/article.stories.js]: resolved

Comment (aronshamash) [reply packages/article/article.stories.js]: yes

Comment (colincclark) [reply packages/article/article.stories.js]: seems ok to me - better this way - it is more human readable - the actual tpa template name is not required here

**Pull request merged**